homeassistant/components/modbus/climate.py

```diff
@@ -131,6 +132,15 @@ def __init__(
     async def async_added_to_hass(self):
         """Handle entity which will be added."""
         await self.async_base_added_to_hass()
+        state = await self.async_get_last_state()
+        if state and state.attributes.get(ATTR_TEMPERATURE):
+            self._target_temperature = float(state.attributes[ATTR_TEMPERATURE])
+        else:
+            self._target_temperature = self.max_temp
```

Comment:
Isn't it better to leave the temperature as `None` in this case? I think that's more correct than setting it to the max temperature.

Reply:
Caught red handed, I "stole" this code from 
https://github.com/home-assistant/core/blob/dev/homeassistant/components/generic_thermostat/climate.py L#234-276 and adapted it.

Changed